Travis/js/main.js

```diff
@@ -0,0 +1,154 @@
+// Create an object that has a property that is an array. Log one of the elements of that array.
+	
+	myObj={
+		myObjArray:["one","two","three"]
+	}
+
+	console.log(myObj.myObjArray[2])
+
+// Create an object that has a property that is an object. Log one of the properties of that inner object.
+
+	myObjTwo = { 
+		property1:{
+			otherProp1 : 50,
+			otherProp2	: 60
+		}
+	}
+
+	console.log(myObjTwo.property1.otherProp1)
+
+// Create an object that has a property that is a function (method). Call that method.
+
+	myObjThree = {
+		myFunct:()=>{
+			console.log("MyFunctionIsWorking");
+		}
```

Comment:
as we discussed online today, might be best not to use arrow functions in objects for now.  this answer is solid though